microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
 
-|  | Server-side | Client-side |  
-|:--- |:--- |:--- | 
-| **User-Agent Client Hints** \(recommended\) | `Sec-CH-UA` HTTPS header | `navigator.userAgentData` JavaScript method |  
-| **User-Agent string** \(legacy\) | `User-Agent` HTTPS header | `navigator.userAgent` JavaScript method |  
+Mechanisms for browser detection with code location and identifiers:
 
-Microsoft recommends that you use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] whenever possible for the following reasons.  
+| Mechanism | Code Location |  
+|:--- |:--- | 
+| **Feature Detection** \(recommended\) | Client-side: `navigator.userAgentData.brands` JavaScript method |
+| **User-Agent Client Hints** \(recommended\) | Server-side: `Sec-CH-UA` HTTPS header |
+| | Client-side: `navigator.userAgentData` JavaScript method |  
+| **User-Agent String** \(legacy\) | Server-side:`User-Agent` HTTPS header | 
+| | Client-side: `navigator.userAgent` JavaScript method |  
 
-*   Improve code maintainability.  
-*   Reduce code fragility.  
-*   Reduce code breakage from changes to the User-Agent string.  
-    
-When [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable and you must use user agent detection, use the following format to detect Microsoft Edge user agent on Windows and Android.  
+This article describes the methods Microsoft Edge supports for retrieving user agent information.
 
-## User-Agent Client Hints  
+## Feature detection
 
-Starting in Microsoft Edge version 90, access browser information in a cleaner, more privacy-preserving way.  
+Microsoft recommends detecting if a feature is supported in your browser. We do not recommend that you use feature detection to detect a specific browser. For more information see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].
 
-### User-Agent Client Hints HTTPS Header  
+If you must detect browsers, we recommend using client hints as follows.
+
+## User Agent Client Hints
+
+Microsoft Edge supports User-Agent Client Hints starting with version 90.
+
+User-Agent Client Hints easily retrieve basic data such as the browser name and whether or not it is mobile. Also, hints are more likely to be supported in the long term as User-Agent string usage declines.
+
+Employ User-Agent Client Hints when you want to:
+- Determine if new browser activity is from the expected user.
+- Customize hints or instructions if the user is new to this site.
+
+Do not use User-Agent Client Hints to:
+- Block *unsupported* browsers.
+- Restrict access to features on your site.
+
+### User Agent Client Hints HTTPS header
+
+When a browser requests a web page from a server, it sends an HTTPS client header request. The server replies with a specific set of HTTPS response header data. User-Agent Client Hints automatically return the brand and mobile attributes. It can easily include additional data.

```

Comment:
> When a browser requests a web page from a server, it sends an HTTPS client header request

Since we're specifically talking about Edge here, consider something like this phrasing:

> When Microsoft Edge sends an HTTPS request to a server, it sends the User-Agent Client Hints headers that correspond to browser brand and mobile information.

Reply:
Were you going to make this change? You could keep "browser" instead of "Microsoft Edge" if you'd like, but I think the rest of my suggestion is a bit more accurate.

Reply:
Missed it somehow, fixed now!